NOTICE

```diff
@@ -0,0 +1,6 @@
+Apache Flink
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
I wouldn't consider the documentation a distribution and hence we don't need a NOTICE file.

If we do add it then we should follow a similar pattern as we do in the main repo and list all shipped dependencies, covering images and javascript bits.